src/metrics.ts

```diff
@@ -103,51 +122,27 @@ export default class Metrics {
         }
     }
 
-    private startTimer(timeout?: number) {
+    private startTimer(): void {
         if (this.disabled) {
-            return false;
+            return;
```

Comment:
Can remove this guard. startTimer will never happen if metrics are disabled, protected inside start().